draft-ietf-teep-protocol.md

```diff
@@ -438,6 +441,42 @@ versions
   If this field is not present, it is to be treated the same as if
   it contained only version 0.
 
+attestation-payload-format
+: The attestation-payload-format parameter indicates the IANA Media Type of the
+  attestation-payload parameter, where media type parameters are permitted after
+  the media type.  For protocol version 0, the absence of this parameter indicates that
+  the format is "application/eat+cwt; eat_profile=https://datatracker.ietf.org/doc/html/draft-ietf-teep-protocol-12" (see {{I-D.ietf-rats-eat-media-type}}
+  for further discussion).
+  (RFC-editor: upon RFC publication, replace URI above with
+  "https://www.rfc-editor.org/info/rfcXXXX" where XXXX is the RFC number
+  of this document.)
+  This parameter MUST be present only when the QuearyResponse returned from the
+  TEEP Agent with ERR_ATTESTATION_REQUIRED, the TAM MUST send another QearyRequest
```

Comment:
The "MUST send another..." text should be in the TAM Behavior section rather than the definition of the message.

Reply:
Moved them to the TAM Behavior section.